roles/gpu_operator_deploy_from_operatorhub/tasks/deploy_from_catalog.yml

```diff
@@ -108,6 +89,21 @@
   set_fact:
     startingCSV: "{{ gpu_operator_csv_name }}"
 
+- name: Prepare the GPU Operator namespace (only when deploying outside of openshift-operators namespace)
+  when: deploy_bundle_namespace != "openshift-operators"
+  block:
+  - name: Create the namespace, if it did not exist
+    shell: oc get "ns/{{ deploy_bundle_namespace }}" || oc create ns "{{ deploy_bundle_namespace }}"
+
+  - name: Instantiate the OperatorHub OperatorGroup template
```

Comment:
Do users have to do that stuff as well?

Reply:
GUI users, no (done by OpenShift Console certainly?)
CLI users, yes, when they want to install in a dedicated namespace

I'll add in the main thread the error we observed when this `OperatorGroup` resource was missing